deploy/manifests/dev/us-east-2/tenant/storetheindex/dhstore/deployment.yaml

```diff
@@ -19,10 +19,23 @@ spec:
           resources:
             limits:
               cpu: "3"
-              memory: 10Gi
+              memory: 20Gi
```

Comment:
Let's increase both memories to 28Gi considering the worker node type offers up to 32Gi.